recipes/pdcurses/all/conanfile.py

```diff
@@ -10,7 +10,7 @@ class PDCursesConan(ConanFile):
     topics = ("conan", "pdcurses", "curses", "ncurses")
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://pdcurses.org/"
-    license = "Public Domain", "MITX"
+    license = "Unlicense", "MITX"
```

Comment:
There's missing ones here 

- https://github.com/wmcbrine/PDCurses/tree/master/demos#distribution-status
- https://github.com/wmcbrine/PDCurses/tree/master/x11#distribution-status

Idk what this is https://github.com/wmcbrine/PDCurses/blob/618e0aaa31b4728eb4df78ec4de6c2b873908eda/x11/configure#L11-L12

I think it's this https://spdx.org/licenses/FSFUL.html

```suggestion
    license = "Unlicense", "MITX", "CC-BY-4.0", "GPL", "FSFUL"
```

Reply:
Thank you!!!